tests/tests_listen.yml

```diff
@@ -30,6 +30,8 @@
       package:
         name: lsof
         state: present
+      when:
+        - not 'lsof' in ansible_facts.packages
```

Comment:
I know this makes the test a bit faster, but is it worth complicating the code and making it more fragile because of this optimization?

Reply:
If it's not preferable, I'm removing the line 33 and 34.  I looked into other linux-system-roles and thought it's recommended. :)

Reply:
Note: `when: not 'lsof' in ansible_facts.packages` was removed;